test/stepDefinitions/searchResultSteps.js

```diff
@@ -0,0 +1,120 @@
+
```

Comment:
we can move all these to searchSteps.js  as it's all functionality of the search page there are probably even some reusable steps there too that can be used